src/util/createServerRootMixin.js

```diff
@@ -88,36 +131,35 @@ function augmentInstantSearch(
    * @returns {Promise} result of the search, to save for .hydrate
    */
   search.findResultsState = function(componentInstance) {
-    let _renderToString;
-    try {
-      _renderToString = require('vue-server-renderer/basic');
-    } catch (e) {
-      // error is handled by regular if, in case it's `undefined`
-    }
-    if (!_renderToString) {
-      throw new Error('you need to install vue-server-renderer');
-    }
-
     let app;
+    let renderedApp;
 
     return Promise.resolve()
       .then(() => {
-        app = cloneComponent(componentInstance);
-
-        app.instantsearch.helper = helper;
-        app.instantsearch.mainHelper = helper;
-
-        app.instantsearch.mainIndex.init({
-          instantSearchInstance: app.instantsearch,
-          parent: null,
-          uiState: app.instantsearch._initialUiState,
+        app = cloneComponent(componentInstance, {
+          mixins: [
+            {
+              created() {
+                // eslint-disable-next-line consistent-this
+                renderedApp = this;
```

Comment:
is that not the same as `app` already?

Reply:
I don't remember why I did it but I just confirmed they're the same.